scripts/rabbitmq-env

```diff
@@ -87,8 +87,14 @@ DEFAULT_SCHEDULER_BIND_TYPE="db"
 DEFAULT_DISTRIBUTION_BUFFER_SIZE=128000
 [ "x" = "x$RABBITMQ_DISTRIBUTION_BUFFER_SIZE" ] && RABBITMQ_DISTRIBUTION_BUFFER_SIZE=${DEFAULT_DISTRIBUTION_BUFFER_SIZE}
 
+DEFAULT_MAX_NUMBER_OF_PROCESSES=1048576
+[ "x" = "x$RABBITMQ_MAX_NUMBER_OF_PROCESSES" ] && RABBITMQ_MAX_NUMBER_OF_PROCESSES=="${DEFAULT_MAX_NUMBER_OF_PROCESSES}"
+
+DEFAULT_MAX_NUMBER_OF_ATOMS=5000000
+[ "x" = "x$RABBITMQ_MAX_NUMBER_OF_ATOMS" ] && RABBITMQ_MAX_NUMBER_OF_ATOMS=="${DEFAULT_MAX_NUMBER_OF_ATOMS}"
```

Comment:
`==` seems to be bash-specific and is not available in zsh.

Reply:
I totally missed that, assignment vs comparison

Reply:
ShellCheck catches it, too:

```
In scripts/rabbitmq-env line 91:
[ "x" = "x$RABBITMQ_MAX_NUMBER_OF_PROCESSES" ] && RABBITMQ_MAX_NUMBER_OF_PROCESSES=="${DEFAULT_MAX_NUMBER_OF_PROCESSES}"
                                                                                  ^-- SC1097: Unexpected ==. For assignment, use =. For comparison, use [/[[.


In scripts/rabbitmq-env line 94:
[ "x" = "x$RABBITMQ_MAX_NUMBER_OF_ATOMS" ] && RABBITMQ_MAX_NUMBER_OF_ATOMS=="${DEFAULT_MAX_NUMBER_OF_ATOMS}"
                                                                          ^-- SC1097: Unexpected ==. For assignment, use =. For comparison, use [/[[.
```